zproject/prod_settings_template.py

```diff
@@ -230,8 +230,8 @@
         "display_name": "SAML",
         # Path to a square image file containing a logo to appear at
         # the left end of the login/register buttons for this IDP.
-        # The default of "" results in a text-only button.
-        "display_icon": "",
+        # The default of None results in a text-only button.
+        "display_icon": None,
```

Comment:
We should do this as a commented out string, to make it easier for users to understand how to format he path properly in this context.